neo/test/rawiotest/test_medrawio.py

```diff
@@ -0,0 +1,241 @@
+import unittest
+import numpy as np
+
+from neo.rawio.medrawio import MedRawIO
+
+from neo.test.rawiotest.common_rawio_test import BaseTestRawIO
+
+class TestMedRawIO(BaseTestRawIO, unittest.TestCase, ):
+    rawioclass = MedRawIO
+    entities_to_download = [
+        'med'
+    ]
+    entities_to_test = ['med/sine_waves.medd', 'med/test.medd']
+    
+    def test_close(self):
+    
+        filename = self.get_local_path('med/sine_waves.medd')
+        
+        raw_io1 = MedRawIO(filename, password='L2_password')
+        raw_io1.parse_header()
+        raw_io1.close()
+        
+        raw_io2 = MedRawIO(filename, password='L2_password')
+        raw_io2.parse_header()
+        raw_io2.close()
+        
+        
+    def test_scan_med_directory(self):
```

Comment:
This test causes a Segmentation fault on github actions as well as locally for me on ubuntu 20.04. On what system did this test run successfully for you?